docs/index.bs

```diff
@@ -3257,9 +3275,11 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
                   1. Run the following substeps [=in parallel=]:
                       1. Let |fetchHandlerResponse| be the result of [=Create Fetch Event and Dispatch=] with |request|, |registration|, |useHighResPerformanceTimers|, |timingInfo|, |workerRealm|, |reservedClient|, |preloadResponse|, and |raceResponse|.
                       1. If |fetchHandlerResponse| is not null and not a [=network error=], and |raceFetchController| is not null, [=fetch controller/abort=] |raceFetchController|.
-                      1. [=queue/Enqueue=] |fetchHandlerResponse| to |queue|.
+                      1. [=queue/Enqueue=] |fetchHandlerResponse| as [=race result/race response=] and {{RouterSourceEnum/"fetch-event"}} as [=race result/used route=] to |queue|.
                   1. Wait until |queue| is not empty.
-                  1. Return the result of [=dequeue=] |queue|.
+                  1. Let |result| be the result of [=dequeue=] |queue|.
+                  1. Set |timingInfo|'s [=service worker timing info/worker final router source=] be set to [=result/used route=] in |result|.
```

Comment:
nit but  I guess "[=result/used route=] in |result|" is usually written like " |result|'s [=result/used route=]"